sympy/physics/vector/frame.py

```diff
@@ -720,57 +720,38 @@ def orient_explicit(self, parent, dcm):
         dcm : Matrix, shape(3, 3)
             Direction cosine matrix that specifies the relative rotation
             between the two reference frames.
-
+        rotation matrix : Matrix Rotation Direction
+            The simple matrix rotation relative ``parent`` to ``dcm``.
```

Comment:
This sentence is unclear.

Reply:
In this package we use the word "direction cosine matrix" instead of "rotation matrix",. They are the same thing, so why would we need two?

Reply:
```
def orient_dcm(self, parent, dcm):
        """ Sets the orientation of this reference frame relative to a parent
        reference frame by explicitly setting the direction cosine matrix.

        Parameters
        ==========

        parent: ReferenceFrame
            Reference frame that this reference frame will be rotated relative
            to.
        dcm : Matrix, shape(3, 3)
            Direction cosine matrix that specifies the relative rotation
            between the two reference frames.
        rotation matrix: Direction Cosine Matrix
            Rotate relative to ``parent``.
```
* When I read [reference](https://moorepants.github.io/learn-multibody-dynamics/orientation.html#orientation-simple-successive), I understand that ``B.orient_dcm(A, B_C_A)`` or `B.orient_dcm(A, B_C_A)` means that B (reference frame) is relatively rotated to A (reference frame). We can verify using `B.dcm(A)` .
*  Please let me know if any modifications or corrections are needed for the code.
* Sorry for the spam message. I'm new to open source and apologize for any inconvenience. I'm providing:

Reply:
The docstring implies you have added a new argument, but actually it is unclear.

Reply:
```
    def orient_dcm(self, parent, dcm):
        """Sets the orientation of this reference frame relative to another (parent) reference frame
        using a direction cosine matrix that describes the rotation from the parent to the child.

        Parameters
        ==========

        parent : ReferenceFrame
            Reference frame that this reference frame will be rotated relative
            to.
        dcm : Matrix, shape(3, 3)
            Direction cosine matrix that specifies the relative rotation
            between the two reference frames.

        Warns
        ======

```
I think the current docstring implies that a change is needed. Please update me if any change is need.

Reply:
I am referring to this addition:

```
        rotation matrix: Direction Cosine Matrix
            Rotate relative to ``parent``.
```

Is this a new argument? It is formatted in that way.

Reply:
Okay, could you then remove the `rotation matrix` argument and use something as you propose [here](https://github.com/sympy/sympy/pull/25916/files#r1402041601). Make sure that it is clear that the DCM describes the orientation of the parent with respect to the child in `orient_explicit`.